tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "4.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+      default: "rhtap-releng-tenant/rpa" # This needs to change later
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  results:
+    - name: releasePlanAdmission
+      type: string
+      description: The relative path in the workspace to the stored releasePlanAdmission json
+  steps:
+    - name: create-exodus-conf-file
+      image: quay.io/distribution_relengs/publish-to-dev-portal-test-image
+      env:
+        - name: "RELEASE_PLAN_ADMISSION"
+          value: '$(params.releasePlanAdmission)'
+      script: |
+        #!/bin/bash
+        set -e
+
+        EXODUS_ENV=$(get-resource "releaseplanadmission" "$(params.releasePlanAdmission)" "{.spec.data.exodus.env}")
+        output_file="/etc/exodus-rsync.conf"
+        
+        EXODUS_GW_URL="https://exodus-gw.corp.redhat.com"
+        EXODUS_GW_ENV="live"
+        EXODUS_GW_CERT="exodus-prod-cert.crt"
+        EXODUS_GW_KEY="exodus-prod-privatekey.pem"
+
+        # Check if EXODUS_ENV contains "cdn-qa"
+        if [[ "$EXODUS_ENV" == *"cdn-qa"* ]]; then
+          EXODUS_GW_CERT="exodus-stage-cert.crt"
+          EXODUS_GW_KEY="exodus-stage-privatekey.pem"
+          EXODUS_GW_URL="https://exodus-gw.corp.stage.redhat.com"
+        fi
+
+        # Check if EXODUS_ENV contains "cdn-stage"
+        if [[ "$EXODUS_ENV" == *"cdn-stage"* ]]; then
+          EXODUS_GW_ENV="pre"
+        fi
+
+        cat << EOF > $output_file
+        gwcert: $EXODUS_GW_CERT
+        gwkey: $EXODUS_GW_KEY
+        gwurl: $EXODUS_GW_URL
+        gwenv: $EXODUS_GW_ENV
+
+        logger: file:/proc/1/fd/1
+        loglevel: INFO
+
+        environments:
+        - prefix: exodus
+        EOF
+      volumeMounts:
+        - name: secrets
+          mountPath: "/secrets"
+    - name: create-destination-path
+      image: quay.io/distribution_relengs/publish-to-dev-portal-test-image
+      env:
+        - name: "BINARY_DIR_PATH"
+          value: '$(tasks.extract-binaries-from-image.results.binaries_path)'
```

Comment:
I don't think you can access other tasks' results in a task. Instead, this should be passed as a parameter to the task and provided in the pipeline. Also, it shouldn't be an absolute path, but a relative path inside a workspace.